app/static/css/form.css

```diff
@@ -58,29 +93,21 @@ fieldset {
 }
 
 .section-toggle {
+  font-family: "FontAwesome";
   float: right;
   cursor: pointer;
   margin-top: 10px;
 }
 .section-toggle:after {
-  content: "collapse";
+  content: "\f068";
 }
 .section-toggle.collapsed:after {
-  content: "expand";
+  content: "\f067";

```

Comment:
These characters are not rendering for me, not sure why. Other FontAwesome things like the footer work fine.